jira.sgnl.yaml

```diff
@@ -383,6 +384,112 @@ entities:
         type: String
         indexed: true
 
+  Workspace:
+    # https://developer.atlassian.com/cloud/jira/service-desk/rest/api-group-assets/#api-rest-servicedeskapi-assets-workspace-get.
+    displayName: JiraWorkspace
+    externalId: Workspace
+    description: Workspace entity in Jira
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: workspaceId
+        externalId: workspaceId
+        type: String
+        indexed: true
+        uniqueId: true
+
+  # WARNING: In order to sync Objects, you must specify the objectsQlQuery in the adapterConfig.
```

Comment:
@richardxgao can we default the adapterConfig to a query that will return 200 instead?
The goal of templates is to have a working one for each SoR. So, any user must be able to just use the template as is and start ingesting data.

Reply:
From offline discussions, we've decided to

1) Default the `adapterConfig` to be something like:
```json
{
    "objectsQlQuery": "objectType = INPUT_REQUIRED"
}
```
which will return 200 when syncing `Objects`.

2) Note in the documentation that the customer should change this in the config.

Reply:
@shash-sgnl Maybe we should also note that this config field is only used for `Object` entities, so if they don't care about it they can just leave the field as-is, or not sync the entity at all.